libs/blocks/marquee/marquee.js

```diff
@@ -80,8 +80,16 @@ const decorateImage = (media) => {
 
   const imageLink = media.querySelector('a');
   const picture = media.querySelector('picture');
+  const buttonImage = imageLink && imageLink.querySelector('img[src$=".svg"]');
```

Comment:
I think you need to do a check if the link is to a fragment and opening a modal. Those are key points for this, yeah?

Valid use case for SVG's is to be able to link them, to any url.

Example page:
View the bottom marquee. I have the svg linking to adobe.com.
https://marquee-play-button--milo--adobecom.hlx.page/drafts/rclayton/marquee-play-button

Reply:
Do you think something like this would suffice:
```
  let buttonImage;

  if (imageLink && imageLink.dataset?.modalHash) {
    buttonImage = imageLink.querySelector('img[src$=".svg"]');
  }

  if (buttonImage) {
```

If the link links to anything in the fragments folder, it will get decorated with the correct modal data attributes, so it might be enough to just check that there is a modal hash? Or we could check the modal path? 

Also, this would prevent the overlay from occurring in your example, is that okay? It would default to the current `decorateImage` functionality, and replace the link contents with the image.

Reply:
Yeah. Just thinking about if an author does want to add an SVG under the image for some reason, that it doesn't decorate the image. The video button only decorates when linking to/opening a modal.

@auniverseaway Any concerns with this I might be overlooking? I'm thinking about our conversation related to image linking and this feature.